src/main/java/com/google/sps/servlets/PostCommentServlet.java

```diff
@@ -26,6 +26,7 @@
 import java.util.Date;
```

Comment:
The logic you have in this servlet and the PostAnswerServlet are very similar, consider sharing logic between them!

Reply:
i was planning to make PostAnswers and FetchAnswers into a single file, so I don't know if getting the comment logic might be too much regarding the single responsibility principle

Reply:
You don't have to merge into one servlet, but it might be worth factoring out the common logic into a utility file. Consider it, though I don't want to block on it (though at the minimum, perhaps include a TODO)?

Reply:
oh alright, that sounds good!